doc/sphinx/practical-tools/utilities.rst

```diff
@@ -10,7 +10,9 @@ Coq configuration basics
 Describes the basics of Coq configuration that affect
 running and compiling Coq scripts.  It recommends preferred ways to
 install Coq, manage installed packages and structure your project
-directories for maximum ease of use.
+directories for the greatest ease of use.
+
+Feedback on errors and omissions will be appreciated.
```

Comment:
If you want to include this, we should probably say where feedback can be given.

Reply:
I think I'll leave it out.